3기/choijiyoung/week1/grid/style.css

```
@@ -0,0 +1,145 @@
+@charset "utf-8";
+
+:root {
+  --gray: #d9d9d9;
+}
+
+* {
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 1;
+
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+#app {
+  display: grid;
+  grid-template-columns: 250px auto;
+  grid-template-rows: auto 100px;
+  max-width: 960px;
+  min-width: 320px;
+  height: 900px;
+  margin: 0 auto;
+  gap: 16px;
+}
+
+/* header */
+header {
+  grid-row: span 2;
+  width: 250px;
+  height: 100%;
+  padding: 32px;
+  background-color: var(--gray);
+}
+
+.header__logo {
+  margin: 0;
+}
+
+/* main- posts */
+.posts {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: 1fr 1fr;
+  height: 100%;
+  padding: 32px;
+  gap: 16px;
+  background-color: var(--gray);
+}
+
+.post__item {
+  padding: 32px;
+  background-color: #fff;
+}
+
+.post__item:nth-of-type(1) {
+  grid-row: span 2;
+}
+
+.post__item:nth-of-type(2) {
+  order: -1;
+}
+
+.post__item:nth-of-type(3) {
+  order: -2;
+}
+
+.post__item:nth-of-type(4) {
+  order: 2;
+}
+
+.post__item:nth-of-type(5) {
+  order: 1;
+}
+
+/* footer */
+footer {
+  height: 100px;
+  padding: 32px;
+  background-color: var(--gray);
+}
+
+/* 반응형 - 태블릿 */
+@media screen and (max-width: 1023px) {
+  .posts {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: repeat(3, 1fr);
+    height: 100%;
+    padding: 32px;
+    gap: 16px;
+    background-color: var(--gray);
+  }
+
+  .post__item:nth-of-type(1) {
+    grid-column: span 2;
+    grid-row: initial;
+    order: -2;
+  }
+
+  .post__item:nth-of-type(2) {
+    order: -1;
+  }
+
+  .post__item:nth-of-type(3) {
+    order: 0;
+  }
+
+  .post__item:nth-of-type(4) {
+    order: 1;
+  }
+
+  .post__item:nth-of-type(5) {
+    order: 2;
+  }
+}
+
+/* 반응형 - 모바일 */
+@media screen and (max-width: 767px) {
```

```suggestion
@media (max-width: 767px) {
```

요렇게 생략해도 동일하게 동작해요. 😊